### ISSUE TITLE
did_multipleGT_check

### ISSUE THREAD

**Issue by EdJeeOnGitHub** [EdJeeOnGitHub/did_multipleGT#2]
Create initial check function `did_multipleGT_check()` then move onto `did_multipleGT()`?

**Comment by joel-becker** [comment]
What's your workflow for this @EdJeeOnGitHub? Do you use `usethis` to create these test scripts etc inside RStudio, or is the project loaded up on atom and you copy/paste barbones tests, or what?

**Comment by EdJeeOnGitHub** [comment]
I set up the repo on GitHub and git cloned into a `temp` folder.

Then I created a package in Rstudio using both `usethis::package_whatever()` and `usethis::tests_whatever()` and copied the .git files from the `temp` folder into the folder created by `usethis`.

The route was so roundabout because I made a GitHub project name using underscores which `usethis` won't allow for package names.

**Comment by joel-becker** [comment]
Right, but doesn't this give you all those annoying R accessory files? Do you just add them to .gitignore?
